qir/qsharp/TargetPackages/QSharpVersion/Example.qs

```diff
@@ -1,26 +1,16 @@
-namespace Testing {
```

Comment:
I am not sure why this example was changed (or why a Q# example is checked in in the first place), but I assume that it is not used for testing.

Reply:
This was the example you've sent me which was accidentally checked into the earlier PR which made it to main. This PR just changes it back to whee it was. The `TargetPackages` directory is just an example project that shows how to configure Q# to the different targets. It is not used in testing.